lwallet/Command.hs

```diff
@@ -156,6 +163,33 @@ proposeUpdate =
     lexeme parseSoftwareVersion <*>
     many1 parseProposeUpdateSystem
 
+stakeholderIdP :: Parser StakeholderId
+stakeholderIdP = hash
+
+coinPortionP :: Parser CoinPortion
+coinPortionP = do
+    (token, modifier) <- anyText <&> \s -> case Text.stripSuffix "%" s of
+        Nothing -> (s, identity)
+        Just s' -> (s', (/100))
+    case readMaybe (toString token) of
+        Just (modifier -> a) | a >= 0, a <= 1 -> return $ unsafeCoinPortionFromDouble a
```

Comment:
It would be good to write explicit type at least in one place to make it clearer which numeric type is used. There are many possibilities, any of them should be ok (type application, inline type like `token :: Double`, separate type declaration).